.github/workflows/buildAndTest.yaml

```diff
@@ -88,7 +88,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
         with:
-          java-version: 8
```

Comment:
Should we also valid with Java 11 or is Java 8 support dropped with Ilford?

Reply:
No support was dropped AFAIK, I just figured most integrations would start to use Java 11 so lets start testing against it. Not sure we need integration tests to go against both, though.

Reply:
Fair enough. I forgot we are already (unit) testing against 8, 11, and 15.